install

```diff
@@ -199,6 +206,11 @@ not intentional, please restore the default permissions and try running the
 installer again:
     sudo chmod 775 #{HOMEBREW_PREFIX}
 EOABORT
+# Clone using git:// if TLS 1.2 not built in.
+abort <<-EOABORT if macos_version < "10.9" && !git
+This version of Mac OS X requires git to complete the installation.
+Install Xcode or the Command Line Tools: https://developer.apple.com/download/more/
```

Comment:
Might want to tailor that message for 10.5/10.6 where Xcode/CLT doesn't install Git. Check the Homebrew/brew code for a guide to what specifically to recommend (or I can provide additional pointers).

Reply:
I believe it's only Xcode 3 that doesn't install Git; Xcode 4.2 does when installed on Snow Leopard.

Reply:
It's been an extremely long time but I believe we only ever supported/recommended Xcode 3 on Snow Leopard. Something about the app store or being paid or something.

Reply:
It was with v4 that they switched to the self-contained, store-delivered `/Applications/Xcode.app` arrangement that caused some consternation at the time.

Reply:
Xcode from the app store? How outrageous 😆! Regardless because of 10.5 I reckon it's worth handling the Git case. In an ideal world we recommend CLT when that's available otherwise Xcode otherwise https://git-scm.com/download/mac